### PR TITLE
Update HSC branch to compute covariances with TJPCov

### PR DESCRIPTION
**Pull request by carlosggarcia**
I have merged the tjpcov-related files from master to the HSC branch.

### CODE REVIEW

File: txpipe/twopoint_fourier.py [hunk 1]
```diff
@@ -222,44 +226,57 @@ def load_maps(self):
...
             systmaps_path = pathlib.Path(systmaps_clustering_dir)
             for systmap in systmaps_path.iterdir():
-                systmap_file = str(systmap)
-                print('Reading clustering systematics map file:', systmap_file)
                 try:
                     if systmap.is_file():
-                        syst_map = healpy.read_map(systmap_file, verbose=False)
-                    else:
-                        warnings.warn(f'Map from {systmap} is missing. Skipping...')
+                        if pathlib.Path(systmap).suffix != ".fits":
```
Comment (fjaviersanchez) [inline txpipe/twopoint_fourier.py:242]: I think that this would break if the map is compressed? Maybe add `or pathlib.Path(systmap).suffix != ".fits.gz"`

**Review by fjaviersanchez — APPROVED** [review]
Thanks @carlosggarcia! It looks good to me. There's a potential problem if we save maps in `fits.gz` format, but I think that's not the case so far, so feel free to just merge this, and we can open an issue later if this causes problems (also easier to track/fix just that problem).

File: txpipe/twopoint_fourier.py (hunk 1 above)
Comment (damonge) [reply txpipe/twopoint_fourier.py:242]: There are probably big savings using `fits.gz` for HSC

Comment (carlosggarcia) [reply txpipe/twopoint_fourier.py]: @fjaviersanchez Done. In fact, I would default to `fits.gz` maps.

**Pull request merged**